Mail/Views/Switch User/CreateAccountView.swift

```diff
@@ -0,0 +1,90 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCreateAccount
+import InfomaniakLogin
+import MailCore
+import MailResources
+import SwiftUI
+
+struct CreateAccountView: View {
+    @AppStorage(UserDefaults.shared.key(.accentColor)) private var accentColor = DefaultPreferences.accentColor
+
+    @State private var isPresentingCreateAccount = false
+    @StateObject private var loginHandler = LoginHandler()
+
+    var body: some View {
+        VStack(alignment: .center, spacing: 0) {
```

Comment:
```suggestion
        VStack(spacing: 0) {
```
The alignment is unnecessary as .center is the default value